data/sql/grid_alternatives.sql

```diff
@@ -1468,3 +1468,34 @@ INSERT INTO grid_alternatives(original_grid_name,
                               0,
                               'proj-datumgrid-north-america',
                               NULL, NULL, NULL, NULL);
+ -- Iceland
+
+INSERT INTO grid_alternatives(original_grid_name,
+                              proj_grid_name,
+                              proj_grid_format,
+                              proj_method,
+                              inverse_direction,
+                              package_name,
+                              url, direct_download, open_license, directory)
+                      VALUES ('ISN93_ISN2016.gsb',
+                              'ISN93_ISN2016.gsb',
+                              'NTv2',
+                              'hgridshift',
+                              1,
```

Comment:
```suggestion
                              0,
```
I believe this should be 0, to mean that there is no inversion of direction.